plugins/woocommerce-blocks/assets/js/blocks/single-product/constants.tsx

```diff
@@ -27,7 +27,7 @@ export const DEFAULT_INNER_BLOCKS: InnerBlockTemplate[] = [
 		[
 			[
 				'core/column',
-				{},
+				{ templateLock: false },
```

Comment:
Nit: Isn't the default value of `templateLock` false? In other words, do we need to define it as false explicitly?

Reply:
Nice catch! This attribute is unnecessary in this case. Initially, I thought that, for some reason, the `templateLock` was being modified at some point in the code, preventing users from moving the blocks within the Single Product block. With that in mind, I changed the `templateLock` attribute in multiple places to test that theory. Still, when I noticed the allowed inner blocks were causing it, I removed them from almost all areas and forgot this one 😅 .

I just fixed this in c3f3ae673fc263cfa5c58167f8c61eb4fa747afb